common/src/main/java/io/netty/util/internal/PlatformDependent0.java

```diff
@@ -277,8 +283,74 @@ public Object run() {
             }
 
             UNALIGNED = unaligned;
+
+            Object maybeException = AccessController.doPrivileged(new PrivilegedAction<Object>() {
+                @Override
+                public Object run() {
+                    try {
+                        // Java9 has jdk.internal.misc.Unsafe and not all methods are propergated to
+                        // sun.misc.Unsafe
+                        Class<?> internalUnsafeClass = getClassLoader(PlatformDependent0.class)
+                                .loadClass("jdk.internal.misc.Unsafe");
+                        Method method = internalUnsafeClass.getDeclaredMethod("getUnsafe");
+                        return method.invoke(null);
+                    } catch (ClassNotFoundException e) {
```

Comment:
If the exception is not expected do we want this to prevent the application from starting? It looks like we don't really care what the exception is, we just won't use `jdk.internal.misc.Unsafe`.